addons/registry/2.7.1/tmpl-configmap-velero.yaml

```diff
@@ -10,6 +10,9 @@ data:
     #!/bin/sh
     set -euo pipefail
     echo 'backup starting ...'
+    if [ -z "\$OBJECT_STORE_HOSTNAME" ]; then
```

Comment:
This test will abort the entire script when combined with `set -u`. An empty string _must be substituted_ to satisfy the undefined variable check:

```sh
if [ -z "\${OBJECT_STORE_HOSTNAME-}" ]; then
  export OBJECT_STORE_HOSTNAME=\$OBJECT_STORE_CLUSTER_IP
fi
```

See http://mywiki.wooledge.org/BashFAQ/112 for more details.